cli/tests/test_git_clone.rs

```diff
@@ -599,6 +599,36 @@ fn test_git_clone_remote_default_bookmark(subprocess: bool) {
     revset-aliases.'trunk()' = "feature1@origin"
     "###);
     }
+
+    // A branch with a strange name gets escaped in the config. Windows doesn't like
+    // the strange name.
+    if cfg!(unix) {
+        git_repo.reference("refs/heads/\"", oid, false, "").unwrap();
+        git_repo.set_head("refs/heads/\"").unwrap();
+        let (_stdout, stderr) =
+            test_env.jj_cmd_ok(test_env.env_root(), &["git", "clone", "source", "clone4"]);
+        insta::allow_duplicates! {
+        insta::assert_snapshot!(stderr, @r#"
+    Fetching into new repo in "$TEST_ENV/clone4"
+    bookmark: "@origin        [new] untracked
+    bookmark: feature1@origin [new] untracked
+    bookmark: main@origin     [new] untracked
+    Setting the revset alias "trunk()" to ""\""@origin"
+    Working copy now at: wmwvqwsz abb7c50b (empty) (no description set)
+    Parent commit      : mzyxwzks 9f01a0e0 " feature1@origin main@origin | message
+    Added 1 files, modified 0 files, removed 0 files
+    "#);
```

Comment:
nit: rerun `insta` to update the indent (or split this to separate test function)?

Reply:
Good idea to split it into a separate function.